content/en/about/community/join/index.md

```diff
@@ -45,9 +45,16 @@ Once you've done that, you can head straight to our
 {{< /community_item >}}
 
 {{< community_item logo="./group.svg" alt="Working Groups" >}}
-If you'd like to contribute to the Istio project, learn [how we do things](https://github.com/istio/community/blob/master/README.md) and
-consider participating in our
-[working groups](https://github.com/istio/community/blob/master/WORKING-GROUPS.md).
+If you'd like to contribute to the Istio project, please read about our
+[code of conduct](https://github.com/istio/community/blob/master/CONTRIBUTING.md#code-of-conduct), our
```

Comment:
```suggestion
[code of conduct aggreement](https://github.com/istio/community/blob/master/CONTRIBUTING.md#code-of-conduct) and
```

Reply:
Not sure I like `agreement` (misspelled above).

Reply:
Thanks, I fixed the spelling in the suggested change, above. Don't want to introduce another linter error :)

Reply:
I suppose "agreement" is optional, since "code of conduct" is itself a noun that can stand on its own.